src/mame/atari/a2600.cpp

```diff
@@ -87,20 +96,167 @@ E1 Prog ROM 42 DEMON/DIAMOND (CX2615)
 
 #include "emu.h"
 
-#include "a2600.h"
 
+#include "bus/vcs/compumat.h"
+#include "bus/vcs/dpc.h"
+#include "bus/vcs/harmony_melody.h"
+#include "bus/vcs/rom.h"
+#include "bus/vcs/scharger.h"
+#include "bus/vcs/vcs_slot.h"
+#include "bus/vcs_ctrl/ctrl.h"
+#include "cpu/m6502/m6507.h"
 #include "emupal.h"
 #include "screen.h"
 #include "softlist_dev.h"
+#include "sound/tiaintf.h"
 #include "speaker.h"
+#include "tia.h"
+
+#define USE_NEW_RIOT 0
+
+#if USE_NEW_RIOT
+#include "machine/mos6530n.h"
+#else
+#include "machine/6532riot.h"
+#endif
+
+//#define VERBOSE (LOG_GENERAL)
+#include "logmacro.h"
 
 
-static constexpr auto MASTER_CLOCK_NTSC = 3.579575_MHz_XTAL;
```

Comment:
FYI, there's a commented out line of code on 467 that uses this identifier and is now stale.

Reply:
Yeah, I know. The commented code even did not compile anymore.
That is now fixed as well